components/icons/SearchIcon.tsx

```diff
@@ -7,16 +7,15 @@ interface SearchIconProps {
 const SearchIcon: FC<SearchIconProps> = ({ color }) => {
   return (
     <svg
-      width="20"
-      height="20"
-      viewBox="0 0 20 20"
-      fill="none"
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
```

Comment:
Nice work on resizing all these, I'm sure it was a pain. For future reference, when the change to the SVG's dimensions is just a scaling (preserving aspect ratio), you can just scale the width and height properties, and if you leave viewBox and the SVG path elements the same, they'll automatically scale. The viewBox determines the SVG's internal coordinates, and the width and height are what it renders at.

Reply:
ahh okay gotchu, i will do that next time!